src/main.rs

```diff
@@ -71,12 +71,21 @@ fn main() -> Result<()> {
 
 fn format_file(file_path: &str, dry_run: bool) -> Result<()> {
     let input = File::open(file_path).context("error opening file")?;
-    let mut output = tempfile::NamedTempFile::new()?;
+    let mut output = vec![];
```

Comment:
@zahidkizmaz this would mean you load the whole file into memory again, which we tried to prevent using the tempfile :)

Reply:
yes that is right, i will change this back